eps/README.md

```diff
@@ -0,0 +1,106 @@
+# Enhancement Proposal (EP)
```

Comment:
shall we also update https://github.com/nginxinc/nginx-kubernetes-gateway#contacts with updated advice to not create issues?